bin/local-env/env-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Include useful functions
+. "$(dirname "$0")/includes.sh"
+
+# Check for the site status
+if [[ -z $(curl -s --head --fail $npm_package_config_E2E_URL) ]]; then
+	echo -e $(status_message 'e2e environment is not running, starting it now.')
```

Comment:
Let's capitalize E2E here 👍 
Also, we should maybe end with `...` for consistency with other messaging in similar commands.